CMakeLists.txt

```diff
@@ -0,0 +1,571 @@
+  libMpegTPDec/src/tpdec_drm.cpp
+  libMpegTPDec/src/tpdec_latm.cpp
+  libMpegTPDec/src/tpdec_lib.cpp
+  libMpegTPDec/src/tp_version.h
+  libMpegTPDec/src/tpdec_adif.h
+  libMpegTPDec/src/tpdec_adts.h
+  libMpegTPDec/src/tpdec_drm.h
+  libMpegTPDec/src/tpdec_latm.h)
+
+set(MPEGTPENC_SRC
+  libMpegTPEnc/src/tpenc_adif.cpp
+  libMpegTPEnc/src/tpenc_adts.cpp
+  libMpegTPEnc/src/tpenc_asc.cpp
+  libMpegTPEnc/src/tpenc_latm.cpp
+  libMpegTPEnc/src/tpenc_lib.cpp
+  libMpegTPEnc/src/tp_version.h
+  libMpegTPEnc/src/tpenc_adif.h
+  libMpegTPEnc/src/tpenc_adts.h
+  libMpegTPEnc/src/tpenc_asc.h
+  libMpegTPEnc/src/tpenc_latm.h)
+
+set(PCMUTILS_SRC
+  libPCMutils/src/limiter.cpp
+  libPCMutils/src/pcm_utils.cpp
+  libPCMutils/src/pcmdmx_lib.cpp
+  libPCMutils/src/version.h)
+
+set(SACDEC_SRC
+  libSACdec/src/sac_bitdec.cpp
+  libSACdec/src/sac_calcM1andM2.cpp
+  libSACdec/src/sac_dec.cpp
+  libSACdec/src/sac_dec_conceal.cpp
+  libSACdec/src/sac_dec_lib.cpp
+  libSACdec/src/sac_process.cpp
+  libSACdec/src/sac_qmf.cpp
+  libSACdec/src/sac_reshapeBBEnv.cpp
+  libSACdec/src/sac_rom.cpp
+  libSACdec/src/sac_smoothing.cpp
+  libSACdec/src/sac_stp.cpp
+  libSACdec/src/sac_tsd.cpp
+  libSACdec/src/sac_bitdec.h
+  libSACdec/src/sac_calcM1andM2.h
+  libSACdec/src/sac_dec.h
+  libSACdec/src/sac_dec_conceal.h
+  libSACdec/src/sac_dec_interface.h
+  libSACdec/src/sac_dec_ssc_struct.h
+  libSACdec/src/sac_process.h
+  libSACdec/src/sac_qmf.h
+  libSACdec/src/sac_reshapeBBEnv.h
+  libSACdec/src/sac_rom.h
+  libSACdec/src/sac_smoothing.h
+  libSACdec/src/sac_stp.h
+  libSACdec/src/sac_tsd.h)
+
+set(SACENC_SRC
+  libSACenc/src/sacenc_bitstream.cpp
+  libSACenc/src/sacenc_delay.cpp
+  libSACenc/src/sacenc_dmx_tdom_enh.cpp
+  libSACenc/src/sacenc_filter.cpp
+  libSACenc/src/sacenc_framewindowing.cpp
+  libSACenc/src/sacenc_huff_tab.cpp
+  libSACenc/src/sacenc_lib.cpp
+  libSACenc/src/sacenc_nlc_enc.cpp
+  libSACenc/src/sacenc_onsetdetect.cpp
+  libSACenc/src/sacenc_paramextract.cpp
+  libSACenc/src/sacenc_staticgain.cpp
+  libSACenc/src/sacenc_tree.cpp
+  libSACenc/src/sacenc_vectorfunctions.cpp
+  libSACenc/src/sacenc_bitstream.h
+  libSACenc/src/sacenc_const.h
+  libSACenc/src/sacenc_delay.h
+  libSACenc/src/sacenc_dmx_tdom_enh.h
+  libSACenc/src/sacenc_filter.h
+  libSACenc/src/sacenc_framewindowing.h
+  libSACenc/src/sacenc_huff_tab.h
+  libSACenc/src/sacenc_nlc_enc.h
+  libSACenc/src/sacenc_onsetdetect.h
+  libSACenc/src/sacenc_paramextract.h
+  libSACenc/src/sacenc_staticgain.h
+  libSACenc/src/sacenc_tree.h
+  libSACenc/src/sacenc_vectorfunctions.h)
+
+set(SBRDEC_SRC
+  libSBRdec/src/HFgen_preFlat.cpp
+  libSBRdec/src/env_calc.cpp
+  libSBRdec/src/env_dec.cpp
+  libSBRdec/src/env_extr.cpp
+  libSBRdec/src/hbe.cpp
+  libSBRdec/src/huff_dec.cpp
+  libSBRdec/src/lpp_tran.cpp
+  libSBRdec/src/psbitdec.cpp
+  libSBRdec/src/psdec.cpp
+  libSBRdec/src/psdec_drm.cpp
+  libSBRdec/src/psdecrom_drm.cpp
+  libSBRdec/src/pvc_dec.cpp
+  libSBRdec/src/sbr_crc.cpp
```

Comment:
FWIW; if rebased on top of the latest master, the `sbr_crc.cpp` and `sbr_crc.h` files have been removed upstream.